### ISSUE TITLE
QuickStart Documentation for import not working

### ISSUE THREAD

**Issue by freeweel** [marklogic-community/grove-react-visjs-graph#17]
The documentation says:
import { GraphContainer } from 'grove-react-visjs-graph';

I got an unable to resolve module compile error

I had to type:
import { GraphContainer } from '@marklogic-community/grove-react-visjs-graph';

**Comment by grtjn** [comment]
You are correct. I was aware, but forgot to fix the docs. Thanks for the reminder.. :)
